src/main/java/core/web/filters/AuthenticationFilter.java

```diff
@@ -31,7 +27,10 @@ public void doFilter(ServletRequest servletRequest,
         HttpServletRequest req = (HttpServletRequest) servletRequest;
         HttpServletResponse resp = (HttpServletResponse) servletResponse;
         String url = req.getServletPath();
-        if (url.equals("/login") || url.equals("/drivers/add")) {
+        Set<String> permittedUrls = new HashSet<>();
+        permittedUrls.add("/login");
+        permittedUrls.add("/drivers/add");
```

Comment:
use method 'init' for creating and initializing set

Reply:
sorry forget about your comment yesterday ...